app/controllers/users_controller.rb

```diff
@@ -49,6 +49,18 @@ def destroy
     end
   end
 
+  def following
```

Comment:
follow by RESTful standard, there're only 7 public actions in controller: `index, new, create, show, edit, update, destroy`
Here, you just inserted 2 new other actions: `following` and `followers`
It's not good.
Please create new controller for them two or maybe other solution is also welcomed

Reply:
Yes